src/klein/_plating.py

```diff
@@ -110,6 +261,9 @@ class Plating(object):
 
     CONTENT = "klein:plating:content"
 
+    cooperate = staticmethod(cooperate)
```

Comment:
I don't like this.  Tests have to know to patch this out, which means they need to know how to get to the relevant `Plating` object, and it has to be done with `TestCase.patch` or the like because it's effectively a global.

Reply:
Is it possible to somehow acquire this from a `Klein` instance?  Patching would still be necessary but because `Klein` implements the descriptor protocol, it wouldn't involve patching a global.

It doesn't seem like it can be easily done, though, because the `routed` decorator doesn't have easy access to the `Klein` application.